src/pythermalcomfort/zhang_comfort_model_functions/sensation.py

```diff
@@ -0,0 +1,244 @@
+import numpy as np
+from pythermalcomfort.zhang_comfort_model_functions import coefficients, utilities
+
+def calculate_local_thermal_sensation(
+        t_skin_local_array,
+        dt_skin_local_dt_array,
+        t_skin_local_set_array,
+        t_skin_mean_value,
+        t_skin_mean_set_value,
+        dt_core_local_dt_array,
+        sensation_scale,
+):
+    """Calculate local thermal sensation based on physiological parameters and the selected sensation scale.
+
+    Parameters
+    -----------
+    t_skin_local_array : numpy.ndarray
+        Array of local skin temperatures [°C].
+    dt_skin_local_dt_array : numpy.ndarray
+        Time derivative of local skin temperatures [°C/s].
+    t_skin_local_set_array : numpy.ndarray
+        Array of local skin temperature setpoints [°C].
+    t_skin_mean_value : float
+        Mean skin temperature value [°C].
+    t_skin_mean_set_value : float
+        Setpoint for mean skin temperature [°C].
+    dt_core_local_dt_array : numpy.ndarray
+        Time derivative of core temperatures [°C/s].
+    sensation_scale : str
+        Thermal sensation scale to be used. Accepts either '9-point' or '7-point'.
```

Comment:
please specify which one is used as default

Reply:
Added the comment. The default is "9-point".